common/src/main/java/no/difi/meldingsutveksling/mail/MailSender.java

```diff
@@ -0,0 +1,57 @@
+package no.difi.meldingsutveksling.mail;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
+import org.springframework.stereotype.Component;
+
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import java.util.Properties;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+@RequiredArgsConstructor
+@Slf4j
+@Component
+public class MailSender {
+
+    private final IntegrasjonspunktProperties props;
+
+    public void send(String title, String body) {
+        Properties mailProps = new Properties();
+        mailProps.put("mail.smtp.host", props.getMail().getSmtpHost());
+        mailProps.put("mail.smtp.port", props.getMail().getSmtpPort());
+        mailProps.put("mail.smtp.starttls.enable", "true");
+        mailProps.put("mail.smtp.auth", props.getMail().getEnableAuth());
+
+        String trust = props.getMail().getTrust();
+        if (!isNullOrEmpty(trust)) {
+            mailProps.put("mail.smtp.ssl.trust", trust);
+        }
+
+        Session session = Session.getDefaultInstance(mailProps,
+                new javax.mail.Authenticator() {
+                    protected PasswordAuthentication getPasswordAuthentication() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1161)